Jenkinsfile

```diff
@@ -24,13 +24,14 @@ pipeline {
             steps {
                 withPythonEnv('CPython-3.10') {
                     sh '''
+                        pip3 install pytest-cov pytest-html
                         pip3 install wheel==0.38.*
                         pip3 install numpy==1.23.*
                         pip3 install packaging==23.*
                         pip3 install -e .
                         export MPLBACKEND="agg"
                         export OPENBLAS_NUM_THREADS=1
-                        pytest -v
+                        pytest --cov-report=xml --cov-report term-missing:skip-covered --cov=scilpy
```

Comment:
Remove --cov-report=xml, pytest.ini does the job already. Also, in my opinion, we want to have coverage for the scripts, right ? If so, remove, --cov=scilpy here, .coveragerc will take charge of setting this.